src/reactor.h

```diff
@@ -415,6 +415,15 @@ class Reactor : public cyclus::Facility,
   }
   double longitude;
 
+  // For new Record Time Series functionality
+  #pragma cyclus var { \
+    "default": "RxFuelDemand", \
```

Comment:
Should this be something like FreshFuelDemand?

Reply:
Or ReactorFuelDemand?  I don't think defaults should contain abbreviations. It is harder for new folks to understand